mmrazor/structures/subnet/candidate.py

```diff
@@ -96,7 +96,8 @@ def _format(self, data: _format_input) -> _format_return:
         def _format_item(
                 cond: Union[Dict, Dict[str, Dict]]) -> Dict[str, Dict]:
             """Transform Dict to Dict[str, Dict]."""
-            if isinstance(list(cond.values())[0], dict):
+            if len(cond.values()) > 0 and isinstance(
+                    list(cond.values())[0], dict):
```

Comment:
unness judge `len(cond.values()) > 0`, since the current design of candidate requires non-empty dict when initialization.